src/delphi/eval/token_labelling.py

```diff
@@ -0,0 +1,198 @@
+"""
+This script creates labels for tokens in a sentence. 
+It takes the context of the token into account. 
+Additionally, it can visualize the sentences and their poart-of-speech (POS) tags.
+"""
```

Comment:
This is not a script, but a module - collection of importable functions. They should have their own docstrings, but file docstring is not necessary.